src/app/shared/profile/store/selectors.ts

```diff
@@ -1,13 +1,18 @@
 import { AppState } from '@store';
-import { createSelector } from 'reselect';
-import { ProfileState } from './state';
+import { createSelector } from '@reduxjs/toolkit';
+import { ProfileState } from './reducer';
+import { plainToInstance } from 'class-transformer';
+import { User } from '@shared/user';
 
-const selectFeature = (state: AppState): ProfileState => state.user;
+const selectFeature = (state: AppState): ProfileState => state.profile;
 
 export class ProfileSelectors {
   public static profile = createSelector(
     selectFeature,
-    (state) => state.profile
+    (state) => plainToInstance(
```

Comment:
Let's avoid calling `plainToInstance` in selectors and leave the old implementation - we will refactor the redux approach to avoid having non-serializable state values later